api/controllers/adds_on.js

```diff
@@ -0,0 +1,79 @@
+const { adds_on } = require('../../models/')
+//Addson
+module.exports.getAllAddson = async (req, res) => {
+  const tempAddson = await adds_on.findAll().
+    catch((err) => {
+      console.error(err)
+      throw new Error(err)
+    });
+  return res.response(tempAddson)
+}
+
+async function queryOneAddson(id) {
+  const tempAddson = await adds_on.findOne({
+    where: {
```

Comment:
if the body contains only one line, make it in single line
```suggestion
    where: { id }
```